pkg/plugin/sdk/v1alpha1/plugin/grpc.go

```diff
@@ -74,6 +74,25 @@ func (m *GRPCServer) PostBuildHook(
 		m.Impl.PostBuildHook(req.IsInteractiveMode, prompter)
 }
 
+// PostTestHook translates the gRPC call to the Plugin PostTestHook
```

Comment:
You will have to copy the whole `v1alpha1` package into `v1alpha2` package. `v1alpha1` should be immutable at this point since we already released it.

Reply:
Thinking about this, we should have a test on CI that prevents any changes to previous SDKs. Maybe the best way is to detect changes in `pkg/plugins/sdk/` and only allow for the next release - hardcoded -, in this case, `v1alpha2`.

Reply:
Ive created `v1alpha2`. Are you wanting me to add this test now? Or should I create an issue for it?

Reply:
Addressed in https://github.com/aspect-build/aspect-cli/pull/99.